src/pages/components/button/usage.mdx

```diff
@@ -65,13 +70,13 @@ desired action is to take the user to a new page.
-| _Tertiary_  | For less prominent actions; tertiary buttons can be used in isolation  or paired with a primary button when there are multiple calls to action.               |
-| _Danger_    | For actions that could have destructive effects on the user’s data  (for example, delete or remove). Danger button has three styles: primary, tertiary, and ghost.     |
-| _Ghost_     | For the least pronounced actions; often used in conjunction with a primary button.                                                                            |
+| Variant     | Purpose                                                                                                                                                                                                                                                                                                                          |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Primary_   | For the principal call to action on the page. Primary buttons should only appear once per screen (not including the application header or in a modal dialog).                                                                                                                                                                    |
+| _Secondary_ | For secondary actions on each page. Secondary buttons can only be used  in conjunction with a primary button. As part of a pair, the secondary button's function is to perform the negative action of the set, such as Cancel or Back. Do not use a secondary button in isolation and do not use a button for a positive action. |
+| _Tertiary_  | For less prominent, and sometimes independent, actions. Tertiary buttons can be used in isolation  or paired with a primary button when there are multiple calls to action. Tertiary buttons can also be used for sub-tasks on a page where a primary button for the main and final action is present.                           |
+| _Danger_    | For actions that could have destructive effects on the user’s data  (for example, delete or remove). Danger button has three styles: primary, tertiary, and ghost.                                                                                                                                                               |
+| _Ghost_     | For the least pronounced actions. Ghost buttons are often used in conjunction with a primary button, or with a primary and secondary button pair.                                                                                                                                                                                |
```

Comment:
> or with a primary and secondary button pair

I think this point might need more clarification. I have Friday brain so I can't think of how to clarify it but maybe even just saying in "rare scenarios it can be paired ... ." I think those scenarios mostly happen in progress flow and the ghost button tends to be the "cancel"

Reply:
@aagonzales just want to make sure I've got this right. With the progress flow example, the secondary button would still be a negative action, like Back, and the ghost would be a complete opt-out like Cancel. Is that right?

Reply:
Yup, exactly